unit_tests/speech/test_client.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import base64
+
+from google.cloud._helpers import _to_bytes
+
+_AUDIO_CONTENT = _to_bytes('/9j/4QNURXhpZgAASUkq')
+_B64_AUDIO_CONTENT = base64.b64encode(_AUDIO_CONTENT)
```

Comment:
Please move all non-`unittest` imports into test cases and helpers. See the [blog post][1] by @tseaver on this philosophy.

[1]: https://palladion.com//home/tseaver/obzervationz/2008/unit_testing_notes-20080724